exclip/exclip.c

```diff
@@ -21,17 +21,27 @@
  *    min if val is lower than min, max if val is greater than max, 
  *        and otherwise val
  */
-int clip(int val, int min, int max)
+signed int clip(signed int val,signed int min,signed int max)
 {
-    /* You should modify clip to meet the specifications */
-    return 0;
+if(max>min)
+{
+ if(val>max)
+	return max;
+ else if(val<min)
+	return min;
+ else
+	return val;
+}
+else
```

Comment:
Remove unnecessary `else` keyword. 😄

Reply:
ok i keep that in mind

Reply:
and i have solution of all ex + g + c tasks. i will soon invite you to review my repository means to collab.